src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -72,6 +73,18 @@ public static function substr($string, $from, $length = null)
         return mb_substr($string, $from, $length, $encoding);
     }
 
+    /**
+     * Replaces text within a portion of a string.
+     */
+    public static function substr_replace(string $string, string $replacement, int $start, int $length = null): string
```

Comment:
I'm not sure this actually complies with CS ... should we prefer `substrReplace()` (given `strlenWithoutDecoration` also)

but i understand the reasoning.. i don't mind really :)

Reply:
Maybe, someone like @nicolas-grekas could help us here ?

Reply:
the logic is used once: let's inline it instead of creating a new public method

Reply:
Inlined!